code/api/tests/test_comments.py

```diff
@@ -314,6 +313,111 @@ def test_multi_comments_same_author(self):
+            "jane_smith",
+            "https://github.com/jane_smith",
+            "https://i.imgur.com/k7XVwpB.jpeg"
+        )
+
+        author3 = create_author(
+            204,
+            "Lara Croft",
+            "lara_croft",
+            "https://github.com/lara_croft",
+            "https://i.imgur.com/k7XVwpB.jpeg"
+        )
+
+        comment_author1 = create_comment(
+            1,
+            author1,
+            "text/markdown",
+            "Hello there",
+            post
+        )
+
+        comment_author2 = create_comment(
+            2,
+            author2,
+            "text/markdown",
+            "This is great for testing",
+            post
+        )
+
+        comment_author3 =create_comment(
+            3,
+            author3,
+            "text/markdown",
+            "Wow! This is such a nice comment for testing",
+            post
+        )
+
+        expected_comments = [
+            {
+                "type": "comment",
+                "author": {
+                    "type": "author",
+                    "id": f"http://127.0.0.1:8000/api/author/100",
+                    "host": "http://127.0.0.1:8000/api/",
+                    "displayName": "johnDoe",
+                    "url": f"http://127.0.0.1:8000/api/author/100",
+                    "github": "https://github.com/johnDoe",
+                    "profileImage": "https://i.imgur.com/k7XVwpB.jpeg"
+                },
+                "comment": "Hello there",
+                "contentType": "text/markdown",
+                "published": str(comment_author1.pub_date),
+                "id": f"http://127.0.0.1:8000/api/author/{post.author.id}/posts/{post.id}/comments/1"
+            },
+            {
+                "type": "comment",
+                "author": {
+                    "type": "author",
+                    "id": f"http://127.0.0.1:8000/api/author/200",
+                    "host": "http://127.0.0.1:8000/api/",
+                    "displayName": "jane_smith",
+                    "url": f"http://127.0.0.1:8000/api/author/200",
+                    "github": "https://github.com/jane_smith",
+                    "profileImage": "https://i.imgur.com/k7XVwpB.jpeg"
+                },
+                "comment": "This is great for testing",
+                "contentType": "text/markdown",
+                "published": str(comment_author2.pub_date),
+                "id": f"http://127.0.0.1:8000/api/author/{post.author.id}/posts/{post.id}/comments/2"
+            }
+        ]
+
+        expected = get_comments_json(post, expected_comments, page, size)
+        response = self.client.get(reverse('api:post_comments', args=(post.author.id, post.id)) + f'?page={page}&size={size}')
+
+        self.assertEqual(response.status_code, 200)
+        res_data = json.loads(response.content)
+        
+
         expected_len = len(str(expected))
         res_data_len = len(str(res_data))
         self.assertEqual(expected_len, res_data_len,
```

Comment:
This test fails, as the api returns comments with id 1 and 3. Instead of 1 and 2